src/Dapr.PluggableComponents.Tests/Adaptors/TestServerCallContext.cs

```diff
@@ -0,0 +1,64 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2023 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+using Grpc.Core;
+
+namespace Dapr.PluggableComponents.Adaptors;
+
+internal sealed class TestServerCallContext : ServerCallContext, IDisposable
+{
```

Comment:
Isn't there a class like this defined by gRPC .NET sdk?

Reply:
`ServerCallContext` is `abstract` and the gRPC SDK has it's own implementation, but unfortunately it's not exposed to clients.